### PR TITLE
Drop blob DB in favour of single entities DB

### PR DESCRIPTION
**Pull request by dotfloat**
We do this by merging blob into entities, so any column that used to refer to blob now directly contains a pickled Python object.

### CODE REVIEW

**Comment by oysteoh** [comment]
I've paired up on this and i agree to what has been done! Just a couple of questions - should we fix up the naming of the stuff ( rdb / entites etc... ) and get alembic to work  - or should we have another iteration on that afterwards?

**Comment by dotfloat** [comment]
Renaming can wait, but alembic can't. We should fix it.

File: ert_shared/storage/command.py [hunk 1]
```diff
@@ -17,8 +17,7 @@ def add_parser_options(ap):
         default=False,
         help="Don't create storage_server.json",
     )
-    ap.add_argument("--blob-url", type=str, default=f"sqlite:///{os.getcwd()}/blobs.db")
     ap.add_argument(
-        "--rdb-url", type=str, default=f"sqlite:///{os.getcwd()}/entities.db"
+        "--rdb-url", type=str, default=f"sqlite:///{os.getcwd()}/ert_storage.db"
```
Comment (ManInFez) [inline ert_shared/storage/command.py:21]: should this be not be `--db-url`?

Comment (ManInFez) [reply ert_shared/storage/command.py:21]: or are we keeping the `relational `

Comment (oysteoh) [reply ert_shared/storage/command.py:21]: i think this goes along with my question and @dotfloat 's answer to that was `Renaming can wait, but alembic can't. We should fix it.`

we could discuss if we should also make the naming correct now - or make a new iteration afterwords 👍

Comment (dotfloat) [reply ert_shared/storage/command.py:21]: I didn't like the `--rdb-url` flag at all and I don't think we should have one. I was also uncertain what the name of such a flag should be.

Unlike the naming of the `ert_storage.db`, this flag can be renamed at any point before the API is stable, which is some time from now.

File: ert_shared/storage/extraction_api.py [hunk 2]
```diff
@@ -192,13 +184,13 @@ def _extract_and_dump_update_data(ensemble_id, ensemble_name, rdb_api, blob_api)
...
         if active_observations is not None:
-            active_blob = blob_api.add_blob(active_observations[observation_key])
+            active_blob = active_observations[observation_key]
 
         observation = rdb_api.get_observation(observation_key)
         link = rdb_api._add_observation_response_definition_link(
             observation_id=observation.id,
             response_definition_id=response_definition.id,
-            active_ref=active_blob.id if active_observations is not None else None,
+            active=active_blob if active_observations is not None else None,
```
Comment (xjules) [inline ert_shared/storage/extraction_api.py:193]: Just guessing here, is `active_blob` really a list?

File: tests/storage/test_storage_client.py [hunk 3]
```diff
@@ -31,8 +22,8 @@ def storage_client(server):
 
 
 def test_all_keys(storage_client):
-    names = set([key["key"] for key in storage_client.all_data_type_keys()])
-    assert names == set(["response_one", "response_two", "G:A", "G:B", "group:key1"])
+    names = [key["key"] for key in storage_client.all_data_type_keys()]
```
Comment (ManInFez) [inline tests/storage/test_storage_client.py:25]: Are we guaranteeing the order of these?

Comment (dotfloat) [reply tests/storage/test_storage_client.py]: Changed back to using sets

File: ert_shared/storage/models.py [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+from sqlalchemy.orm import backref, relationship
+from sqlalchemy.orm.collections import attribute_mapped_collection
+from sqlalchemy.schema import UniqueConstraint, MetaData
+from sqlalchemy.sql import func
+
+
+Entity = declarative_base(name="Entity")
+
+
+# class Entity(BaseEntity):
```
Comment (ManInFez) [inline ert_shared/storage/models.py:13]: remove comments

**Review by ManInFez — APPROVED** [review]
:boom:

**Pull request merged**